Dockerfile

```diff
@@ -17,7 +17,26 @@ WORKDIR /todo-app/todo_app
 
 ENTRYPOINT poetry run gunicorn wsgi:start -b 0.0.0.0:80
 
-FROM base as development
+FROM base as dev_base
 RUN poetry install
 
-ENTRYPOINT poetry run flask run --host=0.0.0.0
\ No newline at end of file
+FROM dev_base as development
+ENTRYPOINT poetry run flask run --host=0.0.0.0
```

Comment:
FYI I think only the second entrypoint is actually used, so unless you're expecting the one on line 18 to be executed, we can just get rid of that :)